Makefile

```diff
@@ -1,23 +1,16 @@
 MACKEREL_AGENT_NAME ?= "mackerel-agent"
 MACKEREL_API_BASE ?= "https://mackerel.io"
-MACKEREL_AGENT_VERSION ?= $(shell git describe --tags --abbrev=0 | sed 's/^v//' | sed 's/[-+].*$$//')
+VERSION = $(shell git log --merges --oneline | perl -ne 'if(m/^.+Merge pull request \#[0-9]+ from .+\/bump-version-([0-9\.]+)/){print $$1;exit}')
```

Comment:
I know that. So, I want to declare version explicitly here in future (next pull request).
We can't use git tag, because release build is proceeded before git tagged.
(Build the artifacts, git tag it, and put it in github releases.)